internal/cmd/hana/map.go

```diff
@@ -51,8 +49,8 @@ func NewMapHanaCMD(kymaConfig *cmdcommon.KymaConfig) *cobra.Command {
 
 var (
 	mapCommands = []func(config *hanaCheckConfig) error{
-		createHanaAPIInstance,
-		createHanaAPIBinding,
+		checkAndCreateHanaAPIInstance,
```

Comment:
IMO this prefix of this name say almost nothing. What we check? Maybe we `createHanaAPIInstanceIfNeeded`, `createHanaAPIInstanceIfNotExists` or something like this?